forc/src/ops/forc_init.rs

```diff
@@ -160,7 +160,6 @@ pub fn init(command: InitCommand) -> ForcResult<()> {
     let gitignore_path = Path::new(&project_dir).join(".gitignore");
     // Append to existing gitignore if it exists otherwise create a new one.
     let mut gitignore_file = fs::OpenOptions::new()
-        .write(true)
```

Comment:
For context, Clippy wants this gone because this is implied by the later call to `append`.